VSTS/Tests/L0/SonarQubeScannerMsBuildBegin/DisableAnalysisOnPrBuild.ps1

```diff
@@ -19,6 +20,7 @@ function VerifyPrRun
     Register-Mock IsPrBuild {$IsPrBuild}
     Register-Mock Write-Host
     Register-Mock InvokePreBuildTask
+    Register-Mock Import-Module
```

Comment:
I decided to remove this mock from the Initialize-Test.ps1 script because we only need it in very specific tests.